src/Microsoft.ML.FastTree/GamTrainer.cs

```diff
@@ -34,8 +34,33 @@ namespace Microsoft.ML.Trainers.FastTree
     using SplitInfo = LeastSquaresRegressionTreeLearner.SplitInfo;
 
     /// <summary>
-    /// Generalized Additive Model Learner.
+    /// Generalized Additive Model Trainer.
     /// </summary>
+    /// <remarks>
+    /// <para>
+    /// Generalized Additive Models, or GAMs, model the data as an
+    /// independent set of feature responses similar to a linear model. For each feature, the GAM trainer
+    /// learns a function, known as a shape function, that outputs a response as a function
+    /// of the feature value. (In contrast, a linear model fits a linear response (e.g. a line) to each feature).
+    /// To score an example, the output of each shape function is added up and the total value is the
+    /// output of the model (hence the name Generalized Additive Models).
+    /// </para>
+    /// <para>
+    /// This GAM trainer is implemented using shallow gradient boosted trees (e.g. tree stumps) to learn nonparametric
+    /// shape functions, and is based on the method described in Lou, Caruana, and Gehrke.
+    /// <a href='http://www.cs.cornell.edu/~yinlou/papers/lou-kdd12.pdf'>"Intelligible Models for Classification and Regression."</a> KDD�12, Beijing, China. 2012.
```

Comment:
>� [](start = 137, length = 1)

Might want to XML escape the apostrophe. Replace by:

&apos;

Reply:
Done.